internal/cmd/login.go

```diff
@@ -289,9 +232,24 @@ func loginToInfra(cli *CLI, lc loginClient, loginReq *api.LoginRequest, noAgent
 	}
 
 	fmt.Fprintf(cli.Stderr, "  Logged in as %s\n", termenv.String(loginRes.Name).Bold().String())
+
+	if options.InjectUserSSHConfig {
+		return updateUserSSHConfig(cli)
+	}
+
 	return nil
 }
 
+func equalHosts(x, y string) bool {
+	if x == y {
```

Comment:
I don't think you need this clause because the `strings.TrimPrefix()` part below should catch it anyway.

Reply:
💯  – this code was shuffled around and not added by this PR (unfortunately `git` didn't catch it). I went ahead and included that fix as part of this change!